### ISSUE TITLE
Updating event inserts id and fails "The field 'id' is not a modifiable field"

### ISSUE THREAD

**Issue by gdvalderrama** [nylas/nylas-python#212]
**Describe the bug**

When updating an event obtained using get, the `id` field is automatically inserted, causing the update to fail with:

```
400 Bad Request. Reason: The field 'id' is not a modifiable field.  Please use only the fields mentioned in our docs for PUT requests: https://developer.nylas.com/docs/api/#post/events . Nylas Error Type: invalid_request_error
```

Versions:

python: 3.9.12
nylas: 5.7.0

**To Reproduce**

```
event = nylas.events.where(read_only=False).get("{id}")
event.location = "Some random location"
event.save(notify_participants='false')
```

**Expected behavior**

The event gets updated.

**Additional context**

Some stack trace:

```
NylasApiError: 400 Bad Request. Reason: The field 'id' is not a modifiable field.  Please use only the fields mentioned in our docs for PUT requests: https://developer.nylas.com/docs/api/#post/events . Nylas Error Type: invalid_request_error
  File "sync_data/domain/nylas_client.py", line 89, in update_event
    event.save(notify_participants='false')
  File "nylas/client/restful_models.py", line 772, in save
    super(Event, self).save(**kwargs)
  File "nylas/client/restful_models.py", line 162, in save
    new_obj = self._update_resource(**kwargs)
  File "nylas/client/restful_models.py", line 175, in _update_resource
    return self.api._update_resource(self.cls, self.id, self.as_json(), **kwargs)
  File "nylas/client/client.py", line 699, in _update_resource
    result = self._put_resource(cls, resource_id, data, **kwargs)
  File "nylas/client/client.py", line 694, in _put_resource
    return self._request_update_resource(
  File "nylas/client/client.py", line 685, in _request_update_resource
    result = _validate(response)
  File "nylas/client/client.py", line 70, in _validate
    raise NylasApiError(response)
```

**Comment by mrashed-dev** [comment]
@gdvalderrama Thank you for opening this issue, we are investigating!

**Comment by mrashed-dev** [comment]
@gdvalderrama Thanks again for opening this issue, it should now be fixed in the [latest v5.8.0 release](https://github.com/nylas/nylas-python/pull/216). We've added the missing read only fields for all the API models, including `Event`!
